packages/lightning-lsp-common/src/resources/sfdx/tsconfig-sfdx.base.json

```diff
@@ -2,6 +2,7 @@
     "compilerOptions": {
         "skipLibCheck": true,
         "target": "ESNext",
+        "module": "NodeNext",
```

Comment:
It looks like updating the node version causes existing unit tests failures, which is a WI we are going to support. The latest node version unit tests support is node v18. Can you revert this for the tests to pass by any chance?

Reply:
This is not updating the node version, this is specifying the TypeScript module resolution algorithm. The tests were failing because I missed a line when asserting the contents of the JSON file.